aviary/variable_info/variable_meta_data.py

```diff
@@ -2365,12 +2390,23 @@
 #  \___| \___| \__,_| |_|   |_.__/ \___/ /_\_\
 # ============================================
 
+add_meta_data(
+    Aircraft.Engine.Gearbox.EFFICIENCY,
+    meta_data=_MetaData,
+    historical_name={"GASP": None,
+                     "FLOPS": None,
+                     "LEAPS1": None
+                     },
+    units=None,
+    desc='The efficiency of the gearbox.',
+    default_value=0.98,
+)
 add_meta_data(
     Aircraft.Engine.Gearbox.GEAR_RATIO,
     meta_data=_MetaData,
     historical_name={"GASP": 'INPROP.GR', "FLOPS": None, "LEAPS1": None},
     units=None,
-    desc='The ratio of the RPM_out divided by the RPM_in for the gearbox.',
+    desc='The ratio of the RPM_in divided by the RPM_out for the gearbox.',
```

Comment:
the equivalent gasp variable is propeller rpm/engine rpm

Reply:
I think out/in makes more sense

Reply:
I wanted to deal with numbers larger than one rather than smaller than one. I think this is actually pretty arbitrary as to how we define it since it's a ratio. As long as we're consistent we are ok. I'd like to leave it as it is.

Reply:
For turboprops the convention seems to be to publish reduction gear ratio, which is in/out. To avoid confusion,  add to description something like _"Reduction gear ratio, or the ratio of RPM_in divided by..."_

Also, add comments or to docstring for gearbox builder that this is a model of a _reduction_ gearbox, so gear ratio is in/out

Reply:
In that case, we need to remove INPROP.GR here and update fortran_to_aviary to calculate the inverse of the value

Reply:
done.